website/snippets/_sl-deprecation-notice.md

```diff
@@ -0,0 +1,6 @@
+:::info Deprecation of dbt Metrics and the legacy dbt Semantic Layer
+For users on dbt version 1.5 or earlier &mdash; starting on December 15th, 2023, support for dbt Metrics and the old dbt Semantic Layer (v1.5 and below) will end. To benefit from the latest features, upgrade to the most [recent version](/guides/migration/sl-migration) of dbt and embrace the enhanced dbt Semantic Layer, powered by MetricFlow.
+
+After this date, dbt Labs won't offer support, bug fixes, or security updates for these deprecated features. Documentation for them will be discontinued, and they will be removed from the dbt Cloud user interface.
```

Comment:
The way it's written it could be construed as the user still having access to the features in some manner, it just won't be updated. Suggest clear, concise language that this will be gone.

```suggestion
After December 15th, dbt Labs will no longer support these deprecated features, they will be removed from the dbt Cloud user interface, and their documentation removed from the docs site.
```